src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,127 @@
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        connection = ConnectionUtil.getConnection();
+        String queryCreate = "INSERT INTO manufacturers (`name`, `country`) VALUES (?, ?)";
+        try (PreparedStatement statement = connection.prepareStatement(queryCreate,
+                Statement.RETURN_GENERATED_KEYS)) {
+            statement.setString(1, manufacturer.getName());
+            statement.setString(2, manufacturer.getCountry());
+            statement.executeUpdate();
+            ResultSet resultSet = statement.getGeneratedKeys();
+            if (resultSet.next()) {
+                Long insertedId = resultSet.getLong(1);
+                manufacturer.setId(insertedId);
+            }
+            manufacturer.setDeleted(false);
+            connection.close();
+        } catch (SQLException e) {
+            throw new DataProcessingException("error in create method "
+                    + manufacturer.getName()
+                    + " and "
+                    + manufacturer.getCountry(), e);
+        }
+        return manufacturer;
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long id) throws SQLException {
+        connection = ConnectionUtil.getConnection();
+        String queryGet = "SELECT * FROM manufacturers WHERE id = ? AND is_deleted != true";
+        try (PreparedStatement statement = connection.prepareStatement(queryGet)) {
+            statement.setLong(1, id);
+            try (ResultSet resultSet = statement.executeQuery()) {
+                if (resultSet.next()) {
+                    String name = resultSet.getString("name");
+                    String country = resultSet.getString("country");
+                    Manufacturer manufacturer = new Manufacturer(id, name, country);
+                    manufacturer.setId(id);
+                    connection.close();
+                    return Optional.ofNullable(manufacturer);
+                } else {
+                    connection.close();
+                    return Optional.empty();
+                }
+            } catch (SQLException e) {
+                throw new DataProcessingException("error in get method " + id, e);
+            }
+        }
+    }
+
+    @Override
+    public List<Manufacturer> getAll() {
+        connection = ConnectionUtil.getConnection();
+        String queryGetAll = "SELECT * FROM manufacturers WHERE is_deleted != true";
+        try (Statement statement = connection.createStatement()) {
+            ResultSet resultSet = statement.executeQuery(queryGetAll);
+            List<Manufacturer> manufacturers = new ArrayList<>();
+            while (resultSet.next()) {
+                Long id = resultSet.getLong("id");
+                String name = resultSet.getString("name");
+                String country = resultSet.getString("country");
+                Manufacturer manufacturer = new Manufacturer(id, name, country);
+                manufacturer.setId(id);
```

Comment:
Отримали резалт сет з даними полів з БД. Присвоїли усі значення, окрім isDeleted. У конструкторі не вказана ініціалізація змінної. І знову, присвоюється ID вдруге одним і тим же значенням
```suggestion
                manufacturer.setDeleted(false);
```

Reply:
isDeleted в целом не обязательно сетить, нам вряд ли нужна эта информация